### PR TITLE
updates deps + Jason by default + misc. cleanup

### PR DESCRIPTION
**Pull request by mayel**
supersedes #34 and also closes #32

### CODE REVIEW

File: lib/phoenix_gon/view.ex [hunk 1]
```diff
@@ -72,17 +72,18 @@ defmodule PhoenixGon.View do
...
       {new_key, to_camel_case(value)}
     end
   end
+
   defp to_camel_case(value),
     do: value
 
   defp json_library do
-    Application.get_env(:phoenix_gon, :json_library, Poison)
+    Application.get_env(:phoenix_gon, :json_library, Jason)
```
Comment (mariusbutuc) [inline lib/phoenix_gon/view.ex:87]: ❤️ 💚 💙 💛 💜

**Review by mariusbutuc — COMMENTED** [review]
@mayel @khusnetdinov  Does this PR also resolve #32?

**Comment by mayel** [comment]
Yes, that PR was merged into this too.

**Comment by mariusbutuc** [comment]
Neat! Perhaps [using a keyword to link the two PRs][1] e.g., adding something like

> Closes #32.

in this PR's description can make it easier to close both as soon as this PR is merged in, keeping this repository tidy. 👍 

  [1]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by khusnetdinov** [comment]
@mayel @mariusbutuc PR is ready?

**Comment by mayel** [comment]
@khusnetdinov yes :)

**Comment by khusnetdinov** [comment]
@mayel @mariusbutuc Thanks!

**Pull request merged**